Xamarin.Forms.Core/MultiBinding.cs

```diff
@@ -13,12 +10,12 @@ public sealed class MultiBinding : BindingBase
 	{
 		IMultiValueConverter _converter;
 		object _converterParameter;
-		BindingExpression _expression;
 		IList<BindingBase> _bindings;
 		BindableProperty _targetProperty;
-		bool _isApplying;
-		bool _isCreating;
-		bool _hasSuccessfullyConverted;
```

Comment:
FYI these were mostly to minimize redundant convert's and convertback's which I saw you removed the unit test for.

Reply:
yes, I plan to look at minimising the number of converts, maybe a bit later